main/tests.py

```diff
@@ -82,8 +81,20 @@ def test_root_entrypoint_redirection_logged_user(self, client, create_dog_owner_
         assert response['Location'] == '/homepage/'
 
 
+@pytest.mark.django_db
 class TestHomepageView:
     def test_unlogged_user_access_to_homepage(self, client):
         response = client.get("/homepage/")
         assert response.status_code == 302
         assert response['Location'] == '/login/?next=/homepage/'
+
+    def test_dog_owner_homepage_is_visible_for_dog_owner(self, client, create_dog_owner_user):
+        client.force_login(user=create_dog_owner_user.user)
+        response = client.get("/homepage/")
+        assert response.status_code == 200
+        assert response.context['daycares']
+
+    def test_day_care_homepage_is_visible_for_day_care(self, client, create_daycare_user):
+        client.force_login(user=create_daycare_user.user)
+        response = client.get("/homepage/")
+        assert response.status_code == 200
```

Comment:
is this test relevant for this PR? seems like #64 should take care of it, as it does

Reply:
If I understand it right , according to https://github.com/redhat-beyond/Hotails/pull/67#discussion_r862435378 this test is requierd for now.

Reply:
IMO Yuval is right, this does not seem related to this PR (we rate dealing with dog owner homepage, not day care)

Reply:
Removed this test.